kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/RuntimeDataServiceBase.java

```diff
@@ -653,8 +653,13 @@ public ProcessInstanceCustomVarsList queryProcessesByVariables(String payload, S
             return convertToProcessInstanceCustomVarsList(advanceRuntimeDataService.queryProcessByVariables(convertToServiceApiQueryParam(filter.getAttributesQueryParams()),
                                                                                                             convertToServiceApiQueryParam(filter.getProcessVariablesQueryParams()),
                                                                                                             queryContext));
+        } else if (filter.getOwnersQueryParam() != null) {
```

Comment:
just compute the query param and the execute the new api.
filter.getOwnersQueryParam != null ? filter.getOwnersQueryParam : QueryParam.all(owners)

Reply:
this can't be done as the statement you suggested will result in an `Object` type which is not supported by any method signature

Reply:
I don't undertand.... but if you have an array of owners as list ... you can transform that to QueryParam just checking which one you are getting. If it is query param... then just convert to query param. In the runtime data service you have that signature. Not really sure what you mean about Object as the outcome.

Reply:
Got it, I misunderstood. I think this change is what you suggested. Let me know otherwise